test/WebSites/MvcTagHelpersWebSite/Views/MvcTagHelper_Home/ProductEdit.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@using MvcTagHelpersWebSite.Models
+
+@{
+    var products = (IEnumerable<Product>)ViewBag.Items;
+}
+
+@addtaghelper "Microsoft.AspNet.Mvc.TagHelpers"
+
+<html>
+<head>
+    <meta charset="utf-8" />
+</head>
+<body>
+    @using (@Html.BeginForm("Index", "MvcTagHelper_Product", FormMethod.Post))
+    {
+        @foreach (var product in products)
+        {
+            @await Html.PartialAsync("_ProductPartial", product)
```

Comment:
Many of partial view samples used an underscore for partial view name, and I thought it should be a pattern. Let me know if that is the case.